app/routes/ui/scenario.py

```diff
@@ -77,6 +78,7 @@ async def get(self, scenario_friendly_name, scene_number=0):
                 cur_scene_idx=scene_number_int,
                 scenario_transitions=scenario_obj.transitions,
                 take_screenshot=False,
+                cursor_mode=cursor_mode,
```

Comment:
We lose this information if we complete the scene, so we need to update scene.html's `nextURL` to be something like:
```
nextURL = "/{{ scenario_friendly_name }}/" + next_scene_idx + "?cursor_mode=" + {{ cursor_mode }};
```